src/main/java/uk/gov/hmcts/ccd/domain/service/security/DefaultAuthorisedCaseDefinitionDataService.java

```diff
@@ -53,7 +45,7 @@ public DefaultAuthorisedCaseDefinitionDataService(CaseTypeService caseTypeServic
         CaseTypeDefinition caseTypeDefinition = caseTypeService.getCaseType(caseTypeId);
 
         if (applicationParams.getEnableAttributeBasedAccessControl()
-            && !userHasClassificationForJurisdiction(caseTypeDefinition.getJurisdictionDefinition().getId())) {
+            && !userHasClassificationForJurisdiction(caseTypeDefinition)) {
```

Comment:
We should be able to delete this whole `if` statement as part of this PR now - the method is no longer checking jurisdiction SC but instead against the case type, which already happens a few lines down.

Reply:
agree, removed this